README.md

```diff
@@ -1243,5 +1243,56 @@ C:\path>ruby bin\webpack
 C:\path>ruby bin\webpack-dev-server
 ```
 
+#### Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
+
+If you receive this error ensure your configuration is correct most likely the path to your packs folder is incorrect if you modified from the original path defined.
+
+Example error from `$ ./bin/webpack-dev-server`
```

Comment:
Roger that @gauravtiwari I'll trim it down. I've removed the error and went very specific in the text to ensure no confusion.